tests/MappingTest.php

```diff
@@ -267,11 +268,8 @@ public function testInsertRollback()
                 ]
             ]
         ];
-
+        $this->expectException(SQLException::class);
```

Comment:
The purpose of this test is to ensure inserts are wrapped in a transaction and rolled back in the case of a failure. In this case it's a duplicate `items` ID of 7 which is causing the problem, so presumably the insert into the `customers` table would have succeeded before then.

The test should therefore still ensure that there are no customers with the ID of 3 after the insert fails. The insert itself should be wrapped in a try/catch block with the original assertions made instead of setting up an expectation for `SQLException::class`.